tests/transactions/test_transaction_services.py

```diff
@@ -108,7 +108,7 @@ def test_save_card_with_no_exception(self):
         resp = TransactionServices.save_card(payload)
         self.assertEqual(resp, '. Card details have been saved.')
 
-    @patch('transactions.transaction_services.User.active_objects.filter')
+    @patch('transactions.transaction_services.User.objects.filter')
```

Comment:
@Niyitangasam , why did you opt to switch from `active_objects` to `objects`

Reply:
Hey @Tevinthuku,  I switched because in my implementation I didn't care about using `active_objects` while retrieving users, but I reverted this as it was and changed my implementation to use `active_objects` because it's better to filter active users. Thanks again for the feedback.